msteams-platform/m365-apps/teamsjs-support-m365.md

```diff
@@ -229,6 +229,24 @@ The following table lists host application support for TeamsJS capabilities that
             <td></td>
             <td></td>
         </tr>
+        <tr>
+            <th>search <img src="./images/preview-badge.png" alt="Badge indicating this capability is in preview" /></th>
```

Comment:
The text "search" should link to the search section of this doc, similar to other rows